app/package.json

```diff
@@ -113,7 +113,7 @@
     "react-router-redux": "^5.0.0-alpha.9",
     "react-scrollable-anchor": "^0.5.0",
     "react-transition-group": "^1.2.0",
-    "reactstrap": "^4.8.0",
+    "reactstrap": "5.0.0-beta",
```

Comment:
Although this is in beta it contains some bug fixes from the previous version (in particular, the `Card` components work as expected now) and is the version referenced in the docs.